### DISCUSSION TITLE
make issues nestable

### DISCUSSION THREAD

**Discussion by lastmeta** [community/community#17388] [Projects and Issues]
In the Project board, in board view, I'd like to be able to drag a issue on top of another issue to indicate that it is a subtask of another task. You'd have to handle the case of completing subtasks but not yet completing the master task.

**Comment by mohamed19871908** [comment]
سوف ندقق في الامر

‫في الجمعة، 27 مايو 2022 في 5:37 م تمت كتابة ما يلي بواسطة ‪Jordan Miller‬‏
***@***.***‬‏>:‬
